src/types.ts

```diff
@@ -1,10 +1,4 @@
-export type User = {
-  id?: string;
-  login: string;
-  password: string;
-  age: number;
-  is_deleted: boolean;
-};
+import { User } from "./models/typeORMModels";
```

Comment:
You don't have to import it here if you don't use it here.

Reply:
User class used in 15 row of this file